workspaces/liferay-sample-workspace/client-extensions/liferay-sample-multi-extension-container/package.json

```diff
@@ -0,0 +1,32 @@
+{
+	"dependencies": {
+		"@liferay/js-api": "^0.2.0-pre.0",
+		"react": "16.12.0",
+		"react-dom": "16.12.0",
+		"underscore": "^1.13.6"
+	},
+	"devDependencies": {
+		"@clayui/css": "^3.91.0",
+		"@types/react": "^18.0.33",
+		"@types/react-dom": "^18.0.11",
+		"@types/underscore": "^1.11.4",
+		"babel-loader": "^9.1.2",
+		"copy-webpack-plugin": "^11.0.0",
+		"css-loader": "^6.7.3",
+		"mini-css-extract-plugin": "^2.7.5",
+		"sass": "^1.60.0",
+		"sass-loader": "^13.2.2",
+		"style-loader": "^3.3.2",
+		"ts-loader": "^9.4.2",
+		"typescript": "^5.0.3",
+		"webpack": "^5.78.0",
+		"webpack-cli": "^5.0.1",
+		"webpack-merge": "^5.8.0"
+	},
+	"name": "liferay-sample-multi-extension-container",
+	"scripts": {
+		"build": "webpack --config ./webpack.sharedDeps.js && webpack --config webpack.prod.js",
```

Comment:
IINM, I think you can return multiple webpack configurations in the webpack.config.js file and they are executed one after the other. That way you wouldn't need to invoke `webpack` twice here...

In fact, this won't work in Windows because there's no `&&` operator in the shell...